lib/matplotlib/backends/backend_ps.py

```diff
@@ -3,7 +3,8 @@
 """
 
 import codecs
-import datetime
+from datetime import timezone
+from datetime import datetime
```

Comment:
You should either do like this and then change to `datetime.fromtimestamp` below or only have the previous import and change to `datetime.timezone.utc` below. Same for `backend_svg.py`.

Reply:
I have reverted to the original import statement since datetime.datetime is being used everywhere else and would require to change each occurence